src/shared-components/PaginationControl/PaginationControl.module.css

```
@@ -0,0 +1,104 @@
+
+.pageSizeLabel {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  font-size: 0.875rem;
+  white-space: nowrap;
+}
+
+.pageSizeSelect {
+  background: transparent;
+  border: 1px solidvar(--bs-border-color, #dee2e6);
```

Biome (2.1.2)

[error] Unexpected unknown function: solidvar

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)